src/tic_tac_clojure/game_logic.clj

```diff
@@ -1,16 +1,14 @@
 (ns tic-tac-clojure.game-logic
-  (:require [tic-tac-clojure.board :as board])
-  (:gen-class))
+  (:gen-class)
+  (:require [tic-tac-clojure.board :refer :all]
+            [tic-tac-clojure.validators :refer :all]))
 
-(def winning-indices
-  [[0 1 2]
-  [3 4 5]
-  [6 7 8]
-  [0 3 6]
-  [1 4 7]
-  [2 5 8]
-  [0 4 8]
-  [2 4 6]])
+(defn valid-move?
+  [board num]
+  (and  (in-range? 0 
+                  (- (board-length board) 1) 
```

Comment:
this is nitpicky, but you might want to use a let statement to name the result of `(- (board-length board) 1)` as `last_index` to make it more clear what you're doing